xray/src/build_quadtree.rs

```diff
@@ -208,21 +208,15 @@ pub fn run<T: Extension>(data_provider_factory: DataProviderFactory) {
         .parse::<NodeId>()
         .expect("root_node_id could not be parsed.");
     let parameters = XrayParameters {
+        output_directory,
         point_cloud_client,
         query_from_global: T::query_from_global(&args),
         filter_intervals,
         tile_background_color,
+        tile_size_px,
+        pixel_resolution,
         inpaint_distance_px,
         root_node_id,
     };
-    build_xray_quadtree(
-        output_directory,
-        &Tile {
-            size_px: tile_size,
-            resolution,
-        },
-        &coloring_strategy_kind,
-        &parameters,
-    )
-    .unwrap();
+    build_xray_quadtree(&coloring_strategy_kind, &parameters).unwrap();
```

Comment:
nit: `unwrap` -> `expect("Failed to build xray quadtree.")`?

Reply:
Done